drivers/docker/driver_unix_test.go

```diff
@@ -246,16 +253,16 @@ func TestDockerDriver_Sysctl_Ulimit_Errors(t *testing.T) {
 	for _, tc := range testCases {
 		task, cfg, _ := dockerTask(t)
 		cfg.Ulimit = tc.ulimitConfig
-		require.NoError(t, task.EncodeConcreteDriverConfig(cfg))
+		must.NoError(t, task.EncodeConcreteDriverConfig(cfg))
 
 		d := dockerDriverHarness(t, nil)
 		cleanup := d.MkAllocDir(task, true)
 		t.Cleanup(cleanup)
 		copyImage(t, task.TaskDir(), "busybox.tar")
 
 		_, _, err := d.StartTask(task)
-		require.NotNil(t, err, "Expected non nil error")
-		require.Contains(t, err.Error(), tc.err.Error())
+		must.NotNil(t, err, must.Sprint("Expected non nil error"))
+		must.StrContains(t, err.Error(), tc.err.Error())
```

Comment:
You can swap this out for:

```suggestion
		must.ErrorContains(t, err, tc.err.Error())
```